.storybook/main.js

```diff
@@ -12,4 +12,5 @@ module.exports = {
     '@storybook/addon-docs',
   ],
   framework: '@storybook/react',
+  staticDirs: ['../public'],
```

Comment:
Question: do we need `assets/matrix.svg`? It seems we dont need this one
And we better put the assets at one place.